manifests/init.pp

```diff
@@ -284,84 +284,119 @@
       $package_name_default      = 'nscd'
       $package_source_default    = undef
       $service_name_default      = 'nscd'
-      case $::operatingsystemmajrelease {
-        '5': {
```

Comment:
If you wish so, sure I'll sent pull request tomorrow. Should I revert tests too?


In manifests/init.pp<https://github.com/ghoneycutt/puppet-module-nscd/pull/42#discussion_r58570819>:

> @@ -284,84 +284,119 @@
>        $package_name_default      = 'nscd'
>        $package_source_default    = undef
>        $service_name_default      = 'nscd'
> -      case $::operatingsystemmajrelease {
> -        '5': {


Could you just change the case for EL7 to be '7', '2015', '2016': and adding a comment that the 2015 and 2016 are for Amazon?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ghoneycutt/puppet-module-nscd/pull/42/files/b3fc20c1e5a5f46868951ac9eb8fb6fd60f4a963#r58570819>